src/dmd/expression.d

```diff
@@ -5454,6 +5467,18 @@ extern (C++) final class CallExp : UnaExp
         return new CallExp(loc, e, earg1);
     }
 
+    /***********************************************************
+    * Creates a new function call expression
+    * Params:
+    *       loc   = location
+    *       fd    = the declaration of the function to call
+    *       earg1 = the function argument
+    */
+    static CallExp create(Loc loc, FuncDeclaration fd, Expression earg1)
+    {
+        return new CallExp(loc, fd, earg1);
+    }
```

Comment:
Not really sure why we need this `create` methods and can't let C++ call the constructor, but well that's a matter for another day ...